src/main/java/Functions.java

```diff
@@ -280,17 +280,18 @@ public static void replaceDashes(ArrayList<String> listOfLines, int i) {
     // Inserts code snippets
     public static void codeInsert(String atIndex, ArrayList<String> listOfLines, String guideName, String branch, int i, String position) {
         listOfLines.set(i, listOfLines.get(i).replaceAll("#", ""));
-        for (int x = 0; x < 10; x++) {
-            int y = x + i;
-            if (listOfLines.get(y).startsWith("----")) {
-                listOfLines.set(y, "");
-            }
+
+        if (listOfLines.get(i - 1).startsWith("```") || listOfLines.get(i - 1).startsWith("----")) {
+            listOfLines.set(i - 1, "");
         }
+        if (listOfLines.get(i + 2).startsWith("----")) {
+            listOfLines.set(i + 2, "");
+        }
```

Comment:
@gkwan-ibm what do you think about this? The index is hard coded but that is because the `----` around `#Update...`, `#Replace...` or `#Create...` are always in the same location. This means that we can avoid having a backtick issues in regards to these in the future.